code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -547,12 +562,10 @@
 /datum/sprite_accessory/facial_hair
 
 	icon = 'icons/mob/Human_face.dmi'
-	gender = MALE // barf (unless you're a dorf, dorfs dig chix /w beards :P)

```

Comment:
Not use if removing this actually does anything, but this appears to be an undocumented change.